process_test.go

```diff
@@ -60,7 +60,7 @@ func TestLaunchProcessWithDownloads(t *testing.T) {
 	err = LaunchProcess(cfg, args, &stdout, &stderr)
 	require.NoError(t, err)
 	assert.Equal(t, "", stderr.String())
-	assert.Equal(t, "Preparing auto-download some args\n"+`UPGRADE "chain2" NEEDED at height 49: {"binaries": {"linux/amd64": "https://github.com/regen-network/cosmos-upgrade-manager/raw/auto-download/testdata/repo/zip_binary/autod.zip?checksum=sha256:1517886fb44425c30571316fcbd6380b38f80fb3dbaadc5ecd3303046b43b0cc"}}`+"\n", stdout.String())
+	assert.Equal(t, "Preparing auto-download some args\n"+`UPGRADE "chain2" NEEDED at height 49: {"binaries":{"linux/amd64":"https://github.com/regen-network/cosmos-upgrade-manager/raw/auto-download/testdata/repo/zip_binary/autod.zip?checksum=sha256:1517886fb44425c30571316fcbd6380b38f80fb3dbaadc5ecd3303046b43b0cc"}} module=main`+"\n", stdout.String())
```

Comment:
Thanks for testing this.
I assume there is more in the real log... can you copy the entire log. like:

`ERROR: Preparing...` or whatever the prefix is, to make sure we unit test as close to reality as possible.